### DISCUSSION TITLE
How to access 'localhost:3000'?

### DISCUSSION THREAD

**Discussion by martinotin** [chatwoot/chatwoot#4918] [General]
I have a problem installing chatwoot. After doing all the steps and running 'docker-compose up -d', I access the computer with 'localhost:3000', but it just doesn't work. Please help, thanks

**Comment by vishnu-narayanan** [comment]
Could you check the output of `docker ps` and see if all containers are running?

**Reply by martinotin** [reply]
**It works fine, but when I add in the web server 'localhost:3000', it gives me the following message 'This website cannot be accessed'.**

![image](https://user-images.githubusercontent.com/103229629/175351667-c0a9200b-ad7c-404a-a996-b98eedd67d37.png)

**Reply by pucilpet** [reply]
Silly question - but is this a remote machine?

Localhost is pointing to your local machine. You can use a ssh tunnel to access port 3000 on a remote machine. :)

**Reply by martinotin** [reply]
@pucilpet Obviously that 'localhost:3000' is an example, it is understood logically. Yes, if I am accessing through ssh and at the moment of placing my 'IP:3000' in the web server it does not work.

**Reply by sojan-official** [reply]
@martinotin  if you are using a provider like AWS, they might have firewall rules preventing access to other ports.  You might have to check if that's the case and ensure that access to port 3000 is open. 

https://stackoverflow.com/questions/17161345/how-to-open-a-web-server-port-on-ec2-instance

**Reply by martinotin** [reply]
@sojan-official Thanks for your answer! But my provider is digitalocean

**Reply by martinotin** [reply]
@sojan-official which could be the solution?
